android/app/src/main/java/app/organicmaps/downloader/OnmapDownloader.java

```diff
@@ -178,10 +179,10 @@ private void updateStateInternal(boolean shouldAutoDownload)
     UiUtils.showIf(showFrame, mFrame);
   }
 
-  public OnmapDownloader(MwmActivity activity)
+  public OnmapDownloader(MwmActivity activity, View onMapDownloader)
```

Comment:
```suggestion
  public OnmapDownloader(@NonNull View onMapDownloader)
```

```java
onMapDownloader.getContext()
// or
onMapDownloader.requireContext()
```

Reply:
Ok, I see, MwmActivity is used below
That class sucks
Why did you even touch it?